plugins/amazonq/mynah-ui/src/mynah-ui/ui/main.ts

```diff
@@ -24,7 +24,7 @@ export const createMynahUI = (ideApi: any, featureDevInitEnabled: boolean, codeT
     let mynahUI: MynahUI
     // eslint-disable-next-line prefer-const
     let connector: Connector
-    const messageUserIntentMap = new Map<string, string>()
+    const messageUserIntentMap = new Map<string, string[]>()
```

Comment:
We should change this if we are going to add other metadata. I think something like `responseMetadata` is a better, generic name. We probably want a new type for the data rather than a string list.

Reply:
I will rename this but I am not sure of further data.